src/index.html

```diff
@@ -11,6 +11,39 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="search-form"
+    method="get"
```

Comment:
typically we use uppercase when it comes to methods
```suggestion
    method="GET"
```